tools/wpt/run.py

```diff
@@ -583,6 +583,29 @@ def setup_kwargs(self, kwargs):
         pass
 
 
+class WebKitGTKMiniBrowser(BrowserSetup):
+    name = "webkitgtk_minibrowser"
+    browser_cls = browser.WebKitGTKMiniBrowser
+
+    def install(self, channel=None):
+        raise NotImplementedError
+
+    def setup_kwargs(self, kwargs):
+        if kwargs["binary"] is None:
+            binary = self.browser.find_binary()
+
+            if binary is None:
+                raise WptrunError("Unable to find MiniBrowser binary")
```

Comment:
I notice this doesn't say "in PATH". If it it in PATH, will it be found there?

Reply:
current method doesn't try to find MiniBrowser binary in PATH because its usually installed inside the webkit2gtk-4.0 directory of libexecdir and not in PATH.